### PR TITLE
Migrate to Misk-Web

### PR DESCRIPTION
**Pull request by adrw**
* Move `@misk/` packages and web related Docker/Gradle code to [Misk-Web](https://github.com/square/misk-web)
* [Migration Instructions](https://github.com/square/misk-web)

### CODE REVIEW

File: dependencies.gradle [hunk 1]
```diff
@@ -49,7 +52,22 @@ ext.dep = [
   "logbackClassic": "ch.qos.logback:logback-classic:1.2.3",
   "logbackJsonCore": "ch.qos.logback.contrib:logback-json-core:0.1.5",
   "loggingApi": "io.github.microutils:kotlin-logging:1.4.9",
+  "loggingImpl": "org.apache.logging.log4j:log4j-slf4j-impl:2.10.0",
+  "metricsCore": "io.dropwizard.metrics:metrics-core:4.0.1",
+  "metricsParent": "io.dropwizard.metrics:metrics-parent:4.0.1",
+  "misk": "com.squareup.misk:misk:2018.11.08-a639c2c4",
```
Comment (jjestrel) [inline dependencies.gradle]: Are these changes in the right place? Should misk be depending on itself?

File: misk/build.gradle [hunk 2]
```diff
@@ -10,7 +9,7 @@ kotlin {
 apply plugin: 'org.junit.platform.gradle.plugin'
 apply plugin: 'com.vanniktech.maven.publish'
 apply from: rootProject.file("gradle/localdb.gradle")
-apply from: rootProject.file("gradle/web.gradle")
+apply from: "https://raw.githubusercontent.com/square/misk-web/master/gradle/web.gradle"
```
Comment (jjestrel) [inline misk/build.gradle]: How come we still need the `web` task?

Comment (adrw) [reply misk/build.gradle]: Misk ships with config, example, and loader tabs and still needs web task to compile that code.

File: misk/build.gradle [hunk 3]
```diff
@@ -30,10 +29,6 @@ sourceSets {
   test.java.srcDirs += 'src/test/kotlin/'
   main.resources {
     srcDirs += [
-      'web/@misk/common/lib',
```
Comment (adrw) [inline misk/build.gradle]: @jjestrel These are the parts being abstracted out to Misk-Web

File: misk/build.gradle (hunk 2 above)
Comment (jjestrel) [reply misk/build.gradle]: And both projects need the exact same task without changing any configs?

**Pull request merged**